### ISSUE TITLE
How can we get the discussion of merge request using gitlab4j-api

### ISSUE THREAD

**Issue by ricky001** [gmessner/gitlab4j-api#144]

**Comment by gmessner** [comment]
Use the NotesApi.
```java
GitLabApi gitLabApi = new GitLabApi("http://your.gitlab.server.com", "YOUR_PRIVATE_TOKEN");
List<Note> mergeRequestComments = 
    gitLabApi.getNotesApi().getMergeRequestNotes(projectId, mergeRequestIid);
```

Make sure to use the merge request IID.

**Comment by ricky001** [comment]
Thanks sir, I was using 4.6.0 version so not able to find

**Comment by gmessner** [comment]
@ricky001 
I should have let you know that this functionality was just made available in 4.8.3.
